### ISSUE TITLE
0.17-beta.14 : source maps throw fatal error during `jspm build`

### ISSUE THREAD

**Issue by joeldenning** [jspm/jspm-cli#1804]
Using jspm@0.17-beta.14 and node@5.10.1 produces the following error for me. The command I'm running is `jspm build fetcher - cp-client-auth!sofe - rx sofe/fetcher.js --format amd --skip-rollup`. Unfortunately, I don't have the time right now to figure out exactly what about my project is causing the problem.

```bash
     Creating the single-file build for fetcher - cp-client-auth!sofe - rx...

err  TypeError: Cannot read property 'match' of null
    at /Users/joel/workplace/fetcher/node_modules/systemjs-builder/lib/sourcemaps.js:62:25
    at Array.forEach (native)
    at SourceMapConsumer_eachMapping [as eachMapping] (/Users/joel/workplace/fetcher/node_modules/source-map/lib/source-map-consumer.js:155:14)
    at /Users/joel/workplace/fetcher/node_modules/systemjs-builder/lib/sourcemaps.js:61:24
    at Array.forEach (native)
    at exports.concatenateSourceMaps (/Users/joel/workplace/fetcher/node_modules/systemjs-builder/lib/sourcemaps.js:42:19)
    at createOutput (/Users/joel/workplace/fetcher/node_modules/systemjs-builder/lib/output.js:63:21)
    at exports.writeOutputs (/Users/joel/workplace/fetcher/node_modules/systemjs-builder/lib/output.js:150:16)
    at /Users/joel/workplace/fetcher/node_modules/systemjs-builder/lib/builder.js:705:14
    at tryCatcher (/Users/joel/workplace/fetcher/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/joel/workplace/fetcher/node_modules/bluebird/js/release/promise.js:502:31)
    at Promise._settlePromise (/Users/joel/workplace/fetcher/node_modules/bluebird/js/release/promise.js:559:18)
    at Promise._settlePromise0 (/Users/joel/workplace/fetcher/node_modules/bluebird/js/release/promise.js:604:10)
    at Promise._settlePromises (/Users/joel/workplace/fetcher/node_modules/bluebird/js/release/promise.js:683:18)
    at Promise._fulfill (/Users/joel/workplace/fetcher/node_modules/bluebird/js/release/promise.js:628:18)
    at Promise._resolveCallback (/Users/joel/workplace/fetcher/node_modules/bluebird/js/release/promise.js:423:57)
```

**Comment by joeldenning** [comment]
Note that adding `--skip-source-maps` makes the error go away.

**Comment by davis** [comment]
ah thank you!!

**Comment by guybedford** [comment]
Is it possible get help with a replication here at all? It would be great to look into this further.

**Comment by joeldenning** [comment]
@guybedford looks like in [@piercers duplicate issue](https://github.com/systemjs/plugin-babel/issues/35) that he created [an example repo](https://github.com/piercers/jspm-plugin-babel-10) that duplicates the issue. Have you tried that? If that doesn't work I can give you the code for my project that causes the issue.

**Comment by guybedford** [comment]
Sure, thanks I'll have a look at that.

**Comment by guybedford** [comment]
Ok for some reason Babel started outputting sourceless mappings in the source map. I've filtered these in https://github.com/systemjs/builder/commit/6ba5476e9e378f24ac98e96040adc4599c229479 and can confirm it's working.

**Comment by guybedford** [comment]
Released in jspm 0.17.0-beta.15.
